R/convGDX2MIF.R

```diff
@@ -5,26 +5,28 @@
 #'
 #'
 #' @param gdx a GDX as created by readGDX, or the file name of a gdx
-#' @param gdx_ref reference-gdx for policy costs, a GDX as created by readGDX, or the file name of a gdx
+#' @param gdx_ref reference-gdx for < cm_startyear, used for fixing the prices to this scenario
 #' @param file name of the mif file which will be written, if no name is
 #' provided a magpie object containing all the reporting information is
 #' returned
 #' @param scenario scenario name that is used in the *.mif reporting
 #' @param t temporal resolution of the reporting, default:
 #' t=c(seq(2005,2060,5),seq(2070,2110,10),2130,2150)
+#' @param gdx_refpolicycost reference-gdx for policy costs, a GDX as created by readGDX, or the file name of a gdx
 #' @author Lavinia Baumstark
 #' @examples
 #'
-#' \dontrun{convGDX2MIF(gdx,gdx_ref,file="REMIND_generic_default.csv",scenario="default")}
+#' \dontrun{convGDX2MIF(gdx,gdx_refpolicycost,file="REMIND_generic_default.csv",scenario="default")}
 #'
 #' @export
 #' @importFrom gdx readGDX
 #' @importFrom magclass mbind write.report
 
 convGDX2MIF <- function(gdx, gdx_ref = NULL, file = NULL, scenario = "default",
```

Comment:
why do we need gdx_ref still for price correction?

Reply:
Because prices are Moving Averages, and if `cm_startyear` = 2025, and then the policy scenario calculates its 2020 price values, this takes into account the 2025 value which differs from its reference scenario, and so the reported prices would differ in 2020 even though they should not. See the long debates in https://github.com/pik-piam/remind2/pull/402

Reply:
ah, thanks